compiler/optimizer/OMRSimplifierHelpers.cpp

```diff
@@ -440,6 +440,26 @@ bool branchToFollowingBlock(TR::Node * node, TR::Block * block, TR::Compilation
    return true;
    }
 
+bool fallthroughGoesToBranchBlock(TR::Node *node, TR::Block *block, TR::Compilation *comp)
+   {
+   TR::Block *fallthroughBlock = block->getNextBlock();
+   if (fallthroughBlock == NULL ||
+       fallthroughBlock->isGotoBlock(comp) == false ||
```

Comment:
May I ask you to change this line to `!fallthroughBlock->isGotoBlock(comp)`?

Reply:
Fixed, thanks.